desc/configuration.py

```diff
@@ -1303,78 +1254,30 @@ def body_fun(k_rhok_thetak_Rk_Zk):
 
         return jnp.vstack([rho, theta, phi]).T
 
-    def is_nested(self, nsurfs=10, ntheta=20, nzeta=None, Nt=45, Nr=20):
+    def is_nested(self, grid=None):
```

Comment:
to make this more useful (ie in a callback) it might be useful to have this take R_lmn and Z_lmn as optional args. Default is to use self.R_lmn etc but this allows you to override it